Update README.md (#968)
open-wc/open-wc: packages/es-dev-server/README.md

> Click read more to view different strategies for setting up your project's folder structure.

Reviewer: ```suggestion
Outside of that one requirement, however, `es-dev-server` does not have any opinions on how you should scaffold your project. The following are examples of a variety of different suggested strategies for setting up your project's folder structure.
```

> ### index.html in root

Reviewer: ```suggestion
### index.html in the Root
```

<details>

>   <summary>Read more</summary>

Reviewer: ```suggestion
  <summary>The simplest setup, making sure that all files are accessible, is to place your index.html at the root of your project</summary>
```

> The simplest setup where all files are accessible is to place your index.html at the root of your project:

Reviewer: ```suggestion
Consider this example directory structure in the web root:
```

> If you run `es-dev-server` regularly from the root of this project, you can access your app at `/` or `/index.html` in the browser.

Reviewer: ```suggestion
If you run the `es-dev-server` command from the root of the project, you can access your app at `/` or `/index.html` in the browser.
```

</details>

> ### index.html in a subfoolder

Reviewer: ```suggestion
### index.html in a Subfolder
```

If you move your `index.html` inside a subfolder:

<details>

>   <summary>Read more</summary>

Reviewer: ```suggestion
  <summary>Use the `--open` parameter for when you'd like to keep you index.html in a subfolder.</summary>
```

If you run `es-dev-server` regularly from the root of this project, you can access your app at `/` or `/index.html` in the browser.

</details>

### index.html in a subfoolder

> 

Author: This should go back in to the folded content

If you want your index in a subfolder without this being visible in the browser URL, you can set up a file rewrite rule. [Read more here](#rewriting-file-requests)

</details>

### Monorepos

<details>

>   <summary>Read more</summary>

Reviewer: ```suggestion
  <summary>Use `--app-index` or `--root-dir` when your index.html and web root are in different places, e.g.. in a monorepo setup.</summary>
```

This is the same approach as serving an index.html in a subdirectory, so the section above applies here as well.

</details>

> ### Base element

Reviewer: ```suggestion
### Base Element
```

<details>

>   <summary>Read more</summary>

Reviewer: ```suggestion
  <summary>Use platform features to specify your web root, e.g. in <abbr title="Single Page Applications">SPAs</abbr></summary>
```

You can set up a `<base href="">` element to modify how files are resolved relatively to your index.html. This can be very useful when your index.html is not at the root of your project.

> If you use SPA routing, using a base element is highly recommended. [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base)

Reviewer: ```suggestion
If you use <abbr title="Single Page Application">SPA</abbr> routing, using a base element is highly recommended. [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base)
```
